packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,376 @@ sidebar_position: 3
 
 # Layout transitions
 
-import DocsCompatibilityInfo from '../_shared/_docs_compatibility_info.mdx';
+Layout transition allows you to animate changes in the layout of elements.
```

Comment:
This information doesn't explain what do they do, I would suggest:
```suggestion
Providing a layout transition into an UI element alters each layout change into a smooth transition. A layout change may consist of both changes of size and position and both of them can be animated with layout transition.
```
You can optionally make something bold here.